src/content/docs/new-relic-solutions/get-started/glossary.mdx

```diff
@@ -649,7 +677,13 @@ To learn how New Relic uses events, see [New Relic data types](/docs/data-apis/u
   >
     At New Relic, an integration refers to a connection between a technology and New Relic that allows the reporting of data to New Relic. So, for example, our agents contain various integrations (ways to report data from various app frameworks, or operating systems, or types of databases). Other integrations take the form of a configuration or a procedure (for example, changing a setting or an API endpoint) that allows a service (for example, AWS Lambda or PagerDuty) to send data to us.
 
-    See [all of our integrations](https://newrelic.com/instant-observability/).
+Here's a list of what we consider integrations:
+
+* Agent-based integrations (IAST, AI monitoring, etc)
+* Language-derived integrations
+* Framework-specific integrations (Tomcat, etc.)
+* API based integrations (AWS Lambda, Pagerduty, etc.)
+* Quickstarts to install integrations and contain curated, out-of-the-box experiences (synthetic monitors, alerts, etc.)
```

Comment:
```suggestion
* Quickstarts that install integrations and contain curated, out-of-the-box experiences (synthetic monitors, alerts, etc.)
```

Reply:
Or Quickstarts to install integrations that contain... or Quickstarts that install integrations, which contain... idk